src/model.py

```diff
@@ -131,6 +134,31 @@ def initialize(self, args):
         self._shutdown_event = asyncio.Event()
         self._event_thread.start()
 
+    def _setup_health_check(self):
+        # Check if health check should be enabled
+        self._enable_health_check = (
+            "ENABLE_VLLM_HEALTH_CHECK" in self.model_config["parameters"]
+        ) and (
+            self.model_config["parameters"]["ENABLE_VLLM_HEALTH_CHECK"][
+                "string_value"
+            ].lower()
+            in ["yes", "true"]
+        )
```

Comment:
nit: Can we move these types of checks into a helper function or pb_utils?

ex: 
```python
    def _get_bool_config_param(self, parameter: str) -> bool:
        if parameter in self.model_config["parameters"]
          and (
            self.model_config["parameters"][parameter][
                "string_value"
            ].lower()
            in ["yes", "true"]
        ):
            return True

        return False

...
    self._enable_health_check = self._get_bool_config_param("ENABLE_VLLM_HEALTH_CHECK")
...
    self._report_custom_metrics = self._get_bool_config_param("REPORT_CUSTOM_METRICS")
```

Reply:
Sure, refactored here: [Allow health check to be enableable for all instance counts](https://github.com/triton-inference-server/vllm_backend/pull/73/commits/33367f64b7c0cdcb8002194303edae46a106cc56)